CONTRIBUTING.md

```diff
@@ -5,8 +5,7 @@
 ### Individual contributor
 
 Before we can use your code, you must sign the
-[Google Individual Contributor License Agreement]
-(https://cla.developers.google.com/about/google-individual)
+[Google Individual Contributor License Agreement](https://cla.developers.google.com/about/google-individual)
```

Comment:
We're currently doing DCO instead of CLA. But given that we're still thinking about going back to CLA, it's ok to ffix this proactively.